src/mixins/java/org/spongepowered/common/mixin/core/world/entity/player/PlayerMixin.java

```diff
@@ -170,7 +170,7 @@
 
     @Override
     public boolean bridge$affectsSpawning() {
-        return this.impl$affectsSpawning && !this.shadow$isSpectator() && !this.bridge$isVanishPreventsTargeting();
+        return this.impl$affectsSpawning && (!this.shadow$isSpectator() || !this.bridge$vanishState().untargetable());
```

Comment:
Why was the condition changed ?

nit: Should be use a separate key for "affect spawning" ?